doc/libpmem/pmem_is_pmem.3.md

```diff
@@ -138,11 +138,10 @@ On success, **pmem_unmap**() returns 0. On error, it returns -1 and sets
 
 # NOTES #
 
-On Linux, **pmem_is_pmem**() returns true only if the entire range
-is mapped directly from Device DAX (/dev/daxX.Y) without an intervening
-file system.  In the future, as file systems become available that support
-flushing with **pmem_persist**(3), **pmem_is_pmem**() will return true
-as appropriate.
+On Linux, **pmem_is_pmem**() returned true if the entire range
```

Comment:
I think this should stay in the present tense.

Reply:
Ah, Yes. I will fix it.

Reply:
Fixed.
https://github.com/pmem/pmdk/pull/5088/commits/13e4783524a7a85f88fb15aa90dc9264b650b0af